core/common/src/main/java/alluxio/util/CRC64.java

```diff
@@ -0,0 +1,319 @@
+package alluxio.util;
+
+import java.nio.ByteBuffer;
+import java.util.zip.Checksum;
+
+public class CRC64 implements Checksum {
```

Comment:
Can crc64 not be used as a standard library function? Because different ufs crcs have different algorithms. For example, oss and cos are crc64, s3 is crc32, and hdfs has its own verification. Can we directly connect the verification algorithm of each ufs? Which one is better?